src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -372,7 +374,7 @@ type crSecrets struct {
 
 // TODO: check if resource parameter is really needed
 func (d *Dumper) getIndividualFiles(resource, namespace string, podName, path, location string, tw *tar.Writer) error {
-	args := []string{"-n", namespace, "cp", podName + ":" + path, "/dev/stdout"}
+	args := []string{"-n", namespace, "-c", d.fileContainer, "cp", podName + ":" + path, "/dev/stdout"}
```

Comment:
As discussed, `d.fileContainer` is used for pxc only which means if this function is called in future for PSMDB/PG, this is effectively calling `kubectl -c cp` which means an empty container name and the command will error out.  It would be good to consider this case and fix it, so that getIndividualFiles can work independent of any constraints

Reply:
@cshiv, how do you suggest to handle it? Return error or use default container?

Reply:
@svetasmirnova Using the container name is an assured way and I believe we should continue using container name for getting the file. I would prefer returning an error.

Reply:
Implemented, please review.